third_party/java/arcs/build_defs/internal/tools.oss.bzl

```diff
@@ -15,21 +19,73 @@ def arcs_manifest_parse_test(name, srcs, deps = []):
         if not src.endswith(".arcs"):
```

Comment:
Optional: In the spirit of minimizing validation logic, shall we remove this? 

-- Actually, this is going to be deleted in a different PR, right?

Reply:
Yeah it's getting deleted